src/api/routes/profileRouter.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import profileController from '../controllers/profileController';
+import upload from '../../middlewares/upload';
+import validateInputs from '../../middlewares/validations/body.inputs';
+
+
+const profileRouter = new Router();
+
+profileRouter.patch('/', upload.single('profileImage'), validateInputs(true, 'profile', ['country', 'lastName', 'firstName', 'address', 'gender', 'phoneNumber', 'bio', 'profileImage', 'facebook', 'twitter']),
```

Comment:
minimize the length of this function by declaring an array variable that holds all the value and calls it in the function.